### ISSUE TITLE
Atri Lab Issues🐞 

### ISSUE THREAD

**Issue by money8203** [Atri-Labs/atrilabs-engine#646]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### System configuration

```markdown
1. OS:
2. Bits:
3. Browser:
4. Atri framework version:
5. Virtual environment:
```

**Comment by cruxcode** [comment]
No, it's not a bug. It's the intended behavior as of now. We are going to make live reload feature soon.

**Comment by money8203** [comment]
Oh ok, if you any kind of help I can contribute in the field of frontend part :)

**Comment by cruxcode** [comment]
> Oh ok, if you any kind of help I can contribute in the field of frontend part :)

Sure.
